src/views/Validation.js

```diff
@@ -146,6 +147,7 @@ const Validation = () => {
       </View>
       <Footer>
         <HumbleAlert>{translate('Validation.bodyDoNotReset')}</HumbleAlert>
```

Comment:
Could we change order here and give AppNote more priority and display it as first alert? Now it is as a second one..